### PR TITLE
adding metadata.rb for unicorn as it was missing

### PR DESCRIPTION
**Pull request by phutchins**
I would like to use Berkshelf to source these cookbooks for use on OpsWorks which requires having a metadata.rb in each cookbook.

### CODE REVIEW

File: unicorn/metadata.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+maintainer        "Opscode, Inc."
+maintainer_email  "cookbooks@opscode.com"
+license           "Apache 2.0"
+description       "Manage unicorn"
+version           "0.1"
```
Comment (miketheman) [inline unicorn/metadata.rb:5]: I'm a fan of X.Y.Z - so this should possibly be 0.1.0

Comment (miketheman) [inline unicorn/metadata.rb:5]: But it looks like every other cookbook is always 0.1, so don't mind me. :)

Comment (phutchins) [inline unicorn/metadata.rb:5]: I'm in the same boat with X.Y.Z but went along with their convention... :)

**Comment by jbraeuer** [comment]
Thanks a lot for you pull request. I added the `metadata.rb` for unicorn. As I had to enhance supported OS-list for RedHat-ish distributions, I did not merge the pull request, but created a new commit.

The commit will be visible here, once the code is available in production.

**Comment by phutchins** [comment]
Great. Thanks for putting this in and getting back to me...

-Philip

**Pull request closed without merging**